core/src/main/java/org/elasticsearch/ingest/core/Pipeline.java

```diff
@@ -33,6 +34,7 @@
     final static String DESCRIPTION_KEY = "description";
     final static String PROCESSORS_KEY = "processors";
     final static String ON_FAILURE_KEY = "on_failure";
+    final static String PROCESSOR_TAG_KEY = "processor_tag";
```

Comment:
I thought it would help make it clearer that this is a special config option that is more generic than the options that exist in the processor